### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#35298]
#### VFS team name
Public Websites

#### VFS product name
Outreach & Events Enhancements

#### Point of Contact/Reviewers
Sean Sewell (@seansewell) - Design

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
Date input field is missing label when selecting a specific date. 

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Include a label above the field names for the specific date selection. https://design.va.gov/storybook/?path=/docs/components-date--default If you need some guidance on copy, refer to dates section of design system website. https://design.va.gov/patterns/form-patterns#dates  

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by callen2563** [comment]
@shiragoodman The [design](https://sketch.com/s/8ec18b80-a905-47fb-a217-5720450befa1) has been updated to reflect this change. @srsuddath @tmac9494 need to update in staging
![Outreach   Events - Mobile - Specific date filtered - Error](https://user-images.githubusercontent.com/68560244/150196973-c3de51b9-3f92-4bdd-8312-440076336ffa.png)

**Comment by brianalloyd** [comment]
Group the below tickets into a singular workflow in an effort to make this easier for my FE team to action.
#35295
#35300
#35302
#35303
#35304
Please let me know what questions exist and thanks again for the support.

**Comment by srsuddath** [comment]
This ticket has been accomplished by the following PRs
https://github.com/department-of-veterans-affairs/content-build/pull/958
https://github.com/department-of-veterans-affairs/vets-website/pull/20128
https://github.com/department-of-veterans-affairs/content-build/pull/956
